package.json

```diff
@@ -16,8 +16,8 @@
     "prepare": "husky install",
     "prettify": "prettier --write \"**/*.{ts,js}\"",
     "typecheck": "tsc --noEmit",
-    "cypress:open": "cypress open --config-file tests/cypress/config.json",
-    "cypress:run": "cypress run --config-file tests/cypress/config.json",
+    "cypress:open": "cypress open --config-file tests/cypress/cypress-config.js --e2e --browser chrome",
```

Comment:
@peterwilsoncc The `--browser` parameter was added here to the auto-select browser during the test, otherwise, we have to select the browser from UI during the run test in open mode. 

Also, we can use a script like this `npm run cypress:open -- --browser=firefox` to select the Firefox browser or `npm run cypress:open -- --browser=electron` for Electron.

Should I remove the `--browser` parameter?